cmd/mysterium_fake/mysterium_fake.go

```diff
@@ -58,20 +66,13 @@ func runServer(serverCommand *command_server.CommandRun, waiter *sync.WaitGroup)
 }
 
 func runClient(clientCommand *command_client.CommandRun, waiter *sync.WaitGroup) {
-	err := clientCommand.Run(command_client.CommandOptions{
-		NodeKey:          NODE_KEY,
-		DirectoryRuntime: CLIENT_DIRECTORY_RUNTIME,
-	})
-	if err != nil {
-		fmt.Fprintln(os.Stderr, "Client starting error: ", err)
-		os.Exit(1)
-	}
+	clientCommand.Run()
```

Comment:
Drop unused constants

Reply:
All constants are still in use. Even NodeKey is still used by the node. Refactored constant names to camelcase anyway.

Reply:
Done.